### DISCUSSION TITLE
Print PDF - Family Card

### DISCUSSION THREAD

**Discussion by vikingkarwur** [phili67/ecclesiacrm#1788] [Q&A]
How to generate family card to print as PDF file ? Thanks

**Comment by phili67** [comment]
Sorry for the late feedback.
Could you please be more precise ?
Thanks

**Reply by vikingkarwur** [reply]
I want to display a family card (as a PDF file) which can be generated from the Family Page.

Family Card, consisting of:
- Family name
- Full Address
- & Table of family members (Name, Date of Birth & Customs Fields)

**Reply by vikingkarwur** [reply]
> I want to display a family card (as a PDF file) which can be generated from the Family Page.
> 
> Family Card, consisting of:
> 
> * Family name
> * Full Address
> * & Table of family members (Name, Date of Birth & Customs Fields)

https://2.bp.blogspot.com/-w4T5nbcDmSA/TggdQuvUtfI/AAAAAAAABF8/d3q7HrVLofw/s1600/rekap1.JPG

**Reply by phili67** [reply]
Could you translate the text in english ?
Thank's a lot.
